fetch-in-react/my-app/src/UserDetails.tsx

```diff
@@ -27,8 +27,8 @@ export function UserDetails({ userId, onCancel }: Props) {
       };
       fetchData();
     } catch (err) {
-      alert(err);
-      console.log(setError(err));
+      console.log(alert(err));
```

Comment:
This still displays an alert, which you don't need. It also puts code inside the console.log. This should just be `console.log(err);` (or better, `console.error(err);`).